### PR TITLE
fixed broken links

### PR DESCRIPTION
**Pull request by Goodiec**
This pull request is for this issue https://github.com/mixxxdj/manual/issues/240
Kindly review

### CODE REVIEW

File: source/chapters/livebroadcasting.rst [hunk 1]
```diff
@@ -293,7 +293,7 @@ Method A: Download
 To activate MP3 streaming on macOS, follow these steps:
 
   1. Download
-     `LAME 3.98.4 <https://mega.nz/#!WdwHHTzA!UkdJwUQiihwHb0ShdOBTYj8noSwXluxiKjdWvFQRgOU>`_
+     `LAME 3.98.4 <https://lame.sourceforge.io/links.php#Binaries>`_
      Intel (macOS 10.8+ 64-bit)
```
Comment (Holzhaus) [inline source/chapters/livebroadcasting.rst:297]:
```suggestion
     `LAME <https://lame.sourceforge.io/links.php#Binaries>`_
```

File: source/chapters/library.rst [hunk 2]
```diff
@@ -925,7 +925,7 @@ is required for Windows Vista.
 
 On Linux, :file:`AAC` playback is disabled by default due to licensing restrictions.
 To enable the playback of :file:`AAC` files,
-`build Mixxx from source with m4a/AAC files support <https://mixxx.org/wiki/doku.php/compiling_on_linux#optionalbuild_with_m4a_aac_file_support>`_.
+`build Mixxx from source with m4a/AAC files support <https://github.com/mixxxdj/mixxx/wiki/compiling%20on%20linux#optional-build-with-m4aaac-file-support>`_.
```
Comment (Holzhaus) [inline source/chapters/library.rst:928]:
```suggestion
`build Mixxx from source with m4a/AAC files support <https://github.com/mixxxdj/mixxx/wiki/Compiling-On-Linux#user-content-optional-build-with-m4aaac-file-support>`_.
```

Without this, the sphinx link checker still thinks the link is broken.

File: source/chapters/appendix.rst [hunk 3]
```diff
@@ -443,7 +443,7 @@ Version History
   * Disable track metadata export for .ogg files and TagLib 1.11.1. `lp:1833190 <https://bugs.launchpad.net/mixxx/+bug/1833190>`__
   * Fix interaction of hot cue buttons and looping. `lp:1778246 <https://bugs.launchpad.net/mixxx/+bug/1778246>`__
   * Fix detection of moved tracks. `#2197 <https://github.com/mixxxdj/mixxx/pull/2197>`__
-  * Fix playlist import. `lp:16878282 <https://bugs.launchpad.net/mixxx/+bug/16878282>`__
+  * Fix playlist import. `lp:16878282 <https://bugs.launchpad.net/mixxx/+bug/1687828>`__
```
Comment (Holzhaus) [inline source/chapters/appendix.rst:446]:
```suggestion
  * Fix playlist import. `lp:1687828 <https://bugs.launchpad.net/mixxx/+bug/1687828>`__
```

File: source/chapters/hardware.rst [hunk 4]
```diff
@@ -51,7 +51,7 @@ controllers that Mixxx does not yet support, you can `search the forum
 <https://mixxx.org/forums/search.php?fid[]=7>`_ to see if anyone has
 started a mapping. You can also start one yourself using the information in the
 `Controller Mapping Documentation
-<https://mixxx.org/wiki/doku.php/start#controller_mapping_documentation>`_
+<https://github.com/mixxxdj/mixxx/wiki#controller-mapping-documentation>`_
```
Comment (Holzhaus) [inline source/chapters/hardware.rst:54]:
```suggestion
<https://github.com/mixxxdj/mixxx/wiki#user-content-controller-mapping-documentation>`_
```

**Review by Holzhaus — CHANGES_REQUESTED** [review]
Thanks.

**Comment by Goodiec** [comment]
Done, thank you.

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you.

**Pull request merged**